contracts/service-registry/src/migrations/v_0_4.rs

```diff
@@ -35,7 +36,7 @@ pub fn migrate_services(
         .map(|(service_name, service)| {
             let service = Service {
```

Comment:
```suggestion
            let service = Service {
               coordinator_contract: coordinator_contract.clone(),
               ..service
            };
```

Reply:
this doesn't work, they are different structs and Rust doesn't allow this operation with different structs

Reply:
why are they different structs? the only difference this time around is the coordinator address, isn't it?